DataMigration/src/main/java/org/sagebionetoworks/dian/datamigration/ArcValidationUserCreator.java

```
@@ -0,0 +1,102 @@
+ * that match HM's format, and creates the users on Bridge automatically.
+ *
+ * This is a helper class created by mdephillips to re-create bridge users
+ * every time a new app release is made.
+ */
+public class ArcValidationUserCreator {
+
+    public static void main(String[] args) {
+        System.out.println("Arc Validation User Creation");
+
+        try {
+            SynapseUtil.initializeSynapse();
+
+            // This is the file sage_qa_bridge_setup.zip,
+            // which mdephillips created to automate QA tester creation
+            FileHandleAssociation qaFile = new FileHandleAssociation();
+            qaFile.setAssociateObjectId("syn26235874");
+            qaFile.setFileHandleId("81987693");
+            qaFile.setAssociateObjectType(FileHandleAssociateType.FileEntity);
+
+            // Download and extract the ZIP into JSON files
+            File download = new File("qa_participants.zip");
+            File downloadExt = SynapseUtil.createFolderIfNecessary(
+                    "sage_qa_participant_data");
+            System.out.println("Downloading file");
+            SynapseUtil.synapse.downloadFile(qaFile, download);
+            System.out.println("Unzipping file");
+            UnzipUtil.unzip(download.getAbsolutePath(), downloadExt.getAbsolutePath());
+
+            SynapseUtil.DianParticipantFileFolders participantFolders =
+                    new SynapseUtil.DianParticipantFileFolders();
+            participantFolders.hasdParticipants = new File(
+                    "sage_qa_participant_data/sage_qa_bridge_setup");
+
+            List<SynapseUtil.HmUserData> hmUsers =
+                    SynapseUtil.createHmUserRaterData(participantFolders);
+
+            System.out.println("Accounts ready to create on bridge " + hmUsers.size());
+
+        } catch (IOException | SynapseException e) {
+            System.out.println("Failed to migrate data " + e.getLocalizedMessage());
```

Don't swallow the exceptions, just let them fly.  If there's a problem in production we want the whole pipeline to fail and notify someone to investigate.  Whatever framework we use to run the code will detect failures and send notifications.  This code just has to ensure the exception is raised.